web/locales/he.yml

```diff
@@ -1,79 +1,79 @@
 he:
   TextDirection: 'rtl'
-  Dashboard: Dashboard
-  Status: Status
-  Time: Time
-  Namespace: Namespace
-  Realtime: Real-time
-  History: History
-  Busy: Busy
-  Processed: Processed
-  Failed: Failed
-  Scheduled: Scheduled
-  Retries: Retries
-  Enqueued: Enqueued
-  Worker: Worker
-  LivePoll: Live Poll
-  StopPolling: Stop Polling
-  Queue: Queue
-  Class: Class
-  Job: Job
-  Arguments: Arguments
-  Extras: Extras
-  Started: Started
-  ShowAll: Show All
-  CurrentMessagesInQueue: Current jobs in <span class='title'>%{queue}</span>
-  Delete: Delete
-  AddToQueue: Add to queue
-  AreYouSureDeleteJob: Are you sure you want to delete this job?
-  AreYouSureDeleteQueue: Are you sure you want to delete the %{queue} queue?
-  Queues: Queues
-  Size: Size
-  Actions: Actions
-  NextRetry: Next Retry
-  RetryCount: Retry Count
-  RetryNow: Retry Now
-  Kill: Kill
-  LastRetry: Last Retry
-  OriginallyFailed: Originally Failed
-  AreYouSure: Are you sure?
-  DeleteAll: Delete All
-  RetryAll: Retry All
-  NoRetriesFound: No retries were found
-  Error: Error
-  ErrorClass: Error Class
-  ErrorMessage: Error Message
-  ErrorBacktrace: Error Backtrace
-  GoBack: ← Back
-  NoScheduledFound: No scheduled jobs were found
-  When: When
-  ScheduledJobs: Scheduled Jobs
-  idle: idle
-  active: active
-  Version: Version
-  Connections: Connections
-  MemoryUsage: Memory Usage
-  PeakMemoryUsage: Peak Memory Usage
-  Uptime: Uptime (days)
-  OneWeek: 1 week
-  OneMonth: 1 month
-  ThreeMonths: 3 months
-  SixMonths: 6 months
-  Failures: Failures
-  DeadJobs: Dead Jobs
-  NoDeadJobsFound: No dead jobs were found
-  Dead: Dead
-  Processes: Processes
-  Thread: Thread
-  Threads: Threads
-  Jobs: Jobs
-  Paused: Paused
-  Stop: Stop
-  Quiet: Quiet
-  StopAll: Stop All
-  QuietAll: Quiet All
-  PollingInterval: Polling interval
-  Plugins: Plugins
-  NotYetEnqueued: Not yet enqueued
-  CreatedAt: Created At
-  BackToApp: Back to App
+  Dashboard: לוח מחוונים
+  Status: מצב
+  Time: שעה
+  Namespace: מרחב שם
+  Realtime: זמן אמת
+  History: היסטוריה
+  Busy: עסוקים
+  Processed: מעובדים
+  Failed: נכשלו
+  Scheduled: מתוכננים
+  Retries: נסיונות חוזרים
+  Enqueued: בתור
+  Worker: עובד
+  LivePoll: תשאול חי
+  StopPolling: עצור תשאול
+  Queue: תור
+  Class: מחלקה
+  Job: עבודה
+  Arguments: ארגומנטים
+  Extras: תוספות
+  Started: הותחלו
+  ShowAll: הצג את הכל
+  CurrentMessagesInQueue: עבודות נוכחיות בתור <span class='title'>%{queue}</span>
```

Comment:
@rantav this confused me a bit. You've referred to jobs as מטלות and עבודות.  Are those both correct?  Since things should probably be consistent, which is preferred?